verification/cmd/verification-service/config-docker.yaml

```diff
@@ -0,0 +1,13 @@
+# This config file assumes the service is being run directly from its build
```

Comment:
I thought this config file is to be used when the service is been run from a docker environment??

Reply:
Yup, I fforgot to change the comment when i copied it from `config.yaml`. Thanks.

Reply:
Sure thanks for the ack!